sklearn/base.py

```diff
@@ -535,10 +536,10 @@ class TransformerMixin:
     """Mixin class for all transformers in scikit-learn."""
 
     def fit_transform(self, X, y=None, **fit_params):
-        """Fit to data, then transform it.
-
-        Fits transformer to X and y with optional parameters fit_params
-        and returns a transformed version of X.
+        """
+        Fit to data, then transform it. Fits transformer to X and y with
```

Comment:
I think it is better rendered if you have a short title first (`Fit to data, then transform it`), **then an empty line**, then a more detailed description.